Source/SwiftLintFramework/Rules/Style/VerticalWhitespaceOpeningBracesRule.swift

```diff
@@ -0,0 +1,139 @@
+import Foundation
+import SourceKittenFramework
+
+private extension File {
+    func violatingRanges(for pattern: String) -> [NSRange] {
+        return match(pattern: pattern, excludingSyntaxKinds: SyntaxKind.commentAndStringKinds)
+    }
+}
+
+private extension Dictionary where Key: StringProtocol, Value: StringProtocol {
+    func cleanedKeysDict() -> [String: String] {
```

Comment:
Can we put this somewhere else and share this code instead of duplicating in a few rules?

Reply:
Done. I moved the method to the existing Dictionary+SwiftLint extension file.